action.yml

```diff
@@ -18,6 +18,9 @@ inputs:
   configs:
     description: 'Space separated list of the config IDs.'
     required: true
+  timeout:
+    description: 'How long to wait for the job to finish. Specified as a sequence of decimal numbers with unit suffixes. Default: 2m'
+    required: false
```

Comment:
asi by som tu este pridal `default: 5m` aby s tym by default nebol problem a nemuseli ste to menit vsade.